cpp/example_code/s3/get_put_object_acl.cpp

```diff
@@ -308,27 +316,30 @@ int main()
 
         // If the grantee is by group, then the predefined group URI must 
```

Comment:
This is the first I've heard of group. I fear adding too much information might confuse the reader, so I would stick to canonical user (and ID) throughout and skip the extra information.